### DISCUSSION TITLE
in card mix bindings with html?

### DISCUSSION THREAD

**Discussion by dncpax** [Budibase/budibase#2622] [Help] [answered]
Hi everyone!
So, I' trying to design a simple card where the description is a mix of fields from my data provider. I would like to have some `<br/>` mixed  between values but it's taken literally.
I'm getting:
value 1 \<br/> value 2

What I wanted is:
Title1: value 1
Title2: value 2

Is this possible to do?
Thanks.

**Comment by mjashanks** [comment, marked as answer]
Hey @dncpax .

The easiest way would be to use 2 separate `Paragraph` elements, inside a container, like:

![image](https://user-images.githubusercontent.com/3524181/133679549-0f15b2f9-198b-46e2-a14f-1be0bdcb3e86.png)

**Reply by dncpax** [reply]
Ok, thanks.
